src/formio/templates/editGridRow.ejs

```diff
@@ -1,5 +1,5 @@
   {% for (const key in ctx.flattenedComponents) { %}
-      {% if (ctx.isVisibleInRow(ctx.flattenedComponents[key])) { %}
+      {% if (ctx.isVisibleInRow(ctx.flattenedComponents[key]) && ctx.flattenedComponents[key].label !== "") { %}
```

Comment:
```suggestion
      {% if (ctx.isVisibleInRow(ctx.flattenedComponents[key]) && ctx.flattenedComponents[key].label) { %}
```
this would also account for `undefined` I think?